playground/nuxt.config.ts

```diff
@@ -15,13 +15,15 @@ export default defineNuxtConfig({
     }
   },
   algolia: {
-    // apiKey: process.env.ALGOLIA_API_KEY,
-    // applicationId: process.env.ALGOLIA_APPLICATION_ID,
+    apiKey: process.env.ALGOLIA_API_KEY ?? '599cec31baffa4868cae4e79f180729b',
```

Comment:
I've got these values from an official example: https://codesandbox.io/s/docsearch-v3-debounced-search-gnx87.

I think it's neat to have the playground ready to work without any additional configuration needed from new contributors, and they can still overwrite it if they want to use their own app index.